From aebf6b2645556a5a7c2b4540836710a2432ae366 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Wed, 2 May 2018 07:59:55 -0700
Subject: [PATCH 53/53] Linux 3.18.108

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 488950d29169..cff80c819275 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 18
-SUBLEVEL = 107
+SUBLEVEL = 108
 EXTRAVERSION =
 NAME = Shuffling Zombie Juror
 
-- 
2.14.1

