From 2477ef2dc536dd99b56b48d8155e3f7ba74a3681 Mon Sep 17 00:00:00 2001
From: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Date: Fri, 25 May 2018 10:54:58 +0200
Subject: [PATCH 123/123] Linux 3.18.110

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 2d83b069f06f..70200a6828df 100644
--- a/Makefile
+++ b/Makefile
@@ -1,6 +1,6 @@
 VERSION = 3
 PATCHLEVEL = 18
-SUBLEVEL = 109
+SUBLEVEL = 110
 EXTRAVERSION =
 NAME = Shuffling Zombie Juror
 
-- 
2.14.1

