From 41bf9ae0dc83d16050a470a00fa3a6066cd920a1 Mon Sep 17 00:00:00 2001
From: Henrique Silva <jhenrique09.mcz@hotmail.com>
Date: Mon, 30 Apr 2018 21:06:01 -0300
Subject: [PATCH 5/5] sanders: Use Pixel build fingerprint to bypass safetynet

---
 init/init_sanders.cpp | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/init/init_sanders.cpp b/init/init_sanders.cpp
index 5adbf00..b1686b7 100644
--- a/init/init_sanders.cpp
+++ b/init/init_sanders.cpp
@@ -82,8 +82,8 @@ void vendor_load_properties()
 
     // fingerprint
     property_override("ro.build.description", "sanders-user 7.1.1 NPSS26.116-61-2 7 release-keys");
-    property_override("ro.build.fingerprint", "motorola/sanders/sanders:7.1.1/NPSS26.116-61-2/7:user/release-keys");
-    property_override("ro.vendor.build.fingerprint", "motorola/sanders/sanders:7.1.1/NPSS26.116-61-2/7:user/release-keys");
+    property_override("ro.build.fingerprint", "google/walleye/walleye:8.1.0/OPM1.171019.011/4448085:user/release-keys");
+    property_override("ro.vendor.build.fingerprint", "google/walleye/walleye:8.1.0/OPM1.171019.011/4448085:user/release-keys");
 
     // rmt_storage
     std::string device = GetProperty("ro.boot.device", "");
-- 
2.11.0

